core/dbt/tracking.py

```diff
@@ -412,6 +413,18 @@ def track_partial_parser(options):
     )
 
 
+def track_runnable_timing(options):
+    context = [SelfDescribingJson(RUNNABLE_TIMING, options)]
+    assert active_user is not None, "Cannot track runnable info when active user is None"
+    track(
+        active_user,
+        category="dbt",
+        action="runnable",
```

Comment:
Could we call this `runnable_timing` (or `runnable_task_startup_timing`) for better clarity/discoverability down the line?